src/data/contexts/alertsPromptContext.scss

```diff
@@ -0,0 +1,76 @@
+.alert-card {
+  position: fixed;
+  pointer-events: all;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  border: 1px solid #ccc;
+  border-radius: 20px;
+  cursor: pointer;
+  min-width: 90%;
+  max-width: 500px;
+  padding: 10px 20px;
+  text-align: left;
+  transition: transform 0.3s ease, width 0.3s ease;
+  top: 75%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  word-wrap: break-word;
+}
+
+.alert-content-container {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.alert-icon {
+  font-size: 1.5em;
+  margin-right: 10px;
+  color: inherit;
+  align-self: center;
+}
+
+.alert-header {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+}
+
+.alert-title {
+  font-family: 'Roboto';
```

Comment:
Font family is already set via `body` for the whole app, so you're not supposed to set it here. Look at `_typography.scss`
We also have a variable called `--font-family-default` that's being used in this case.